framework/sandstone_p.h

```diff
@@ -440,6 +488,8 @@ struct SandstoneApplication : public InterruptMonitor, public test_the_test_data
 
     SandstoneBackgroundScan background_scan;
 
+    CpuReschedule *cpu_reschedule;
```

Comment:
the same schedule comment for the variable name. we have a "schedule" for using CPUs, `reschedule()` uses uses it for pick a new one.